cmd/ef_tests/levm/report.rs

```diff
@@ -353,6 +357,88 @@ impl fmt::Display for AccountUpdatesReport {
         for levm_updated_account_only in self.levm_updated_accounts_only.iter() {
             writeln!(f, "  {levm_updated_account_only:#x}:")?;
             writeln!(f, "{}", "    Was updated in LEVM but not in REVM".red())?;
+            let initial_account = self
+                .initial_accounts
+                .get(levm_updated_account_only)
+                .unwrap();
```

Comment:
This can panic if we updated an account in levm that we shouldn't have. Maybe we can do `unwrap_or_default()` but i don't know if that's the right way of solving that.
With the fix I mentioned the end result for that case would be something like
<img width="539" alt="image" src="https://github.com/user-attachments/assets/3dd2fb50-196a-408e-9dbd-aa3b968f0989">
Otherwise that `unwrap()` halts the execution

Reply:
Done in [255a3cc](https://github.com/lambdaclass/ethrex/pull/1280/commits/255a3cc6dbd66ce871c1f910cb159ae6de362842)